### PR TITLE
Helios -> OpaVote

### PR DESCRIPTION
**Pull request by ewdurbin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+The Executive Director will then announce the results to the community.
+At that time, the `🌎 Publish Results` button on OpaVote can be used to
+publish the full results to all voters.
+
+### Achieving Quorum
+
+The PSF Bylaws_ state that one-third (1/3) of the members elligible to vote
+will constitute a quorum. See also:
+
+.. epigraph::
```
Comment (sigmavirus24) [inline README.md]: I think this needs to not be reStructuredText

File: README.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+the voter and clicking "Send Ballot".
+
+If "Automatic reminders" was not selected during election setup,
+OpaVote will automatically send reminders every three days throughout
+the election period.
+
+## Closing an Election
+
+The end of an election requires the EA to verify the results of the election
+with Helios and then present the results to the Executive Director of the PSF.
```
Comment (sigmavirus24) [inline README.md]: Missed a reference to Helios

**Review by sigmavirus24 — COMMENTED** [review]
Love that we're finding new and better solutions

File: README.md (hunk 2 above)
Comment (ewdurbin) [reply README.md]: https://github.com/python/psf-election/pull/6/commits/ac30982030fb061cc6925e3e2315db4f7ea04004

File: README.md (hunk 1 above)
Comment (ewdurbin) [reply README.md]: https://github.com/python/psf-election/pull/6/commits/ac30982030fb061cc6925e3e2315db4f7ea04004

File: README.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+    graffatcolmingov@gmail.com
+    ewdurbin@pyfound.org
+
+The format of the file the election administrator typically gets is of the
+format::
+
+    FirstName,LastName,MembershipType,Email
+
+There is a tool in this repository to handle it called
+``convert-exported-csv-for-opavote.py``. This will generate the necessary
```
Comment (sigmavirus24) [inline README.md:129]: Should this script be in this PR or is that a follow up?

**Review by sigmavirus24 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (ewdurbin) [reply README.md:129]: It'll be added once I have the first report!

**Pull request merged**